shared/menubar/index.render.js.flow

```diff
@@ -1,35 +1,18 @@
 /* @flow */
 import React, {Component} from 'react'
 
-export type RenderProps = {
+import type {Props as FolderProps} from '../folders/render'
+
+export type Props = {
   username: ?string,
-  openKBFS: () => void,
-  openKBFSPublic: (username: ?string) => void,
   logIn: () => void,
-  openKBFSPrivate: (username: ?string) => void,
   showMain: () => void,
   showHelp: () => void,
   showUser: (username: ?string) => void,
   quit: () => void,
-  folders: Array<FolderInfo>,
   debug?: boolean,
   loading: boolean,
   loggedIn: boolean
-}
-
-export default class Render extends Component<void, RenderProps, void> {}
-
-export type FolderInfo = {
-    type: 'folder',
-    folderName: string,
-    isPublic: boolean,
-    isEmpty: boolean,
-    openFolder: () => void
-}
+} & FolderProps
```

Comment:
I don't think I like this pattern because it leads into doing `...this.props` which I think is kinda ugly and vague what is being passed in.

Why not do:

```
...
folderProps: FolderProps,
...
```